### DISCUSSION TITLE
[资讯]强强联合？对空六课将协助虚拟实境系统开发

### DISCUSSION THREAD

**Discussion by March-777th** [share121/inter-knot#5392] [绝区零]
<img width="285" alt="Snipaste_2024-09-25_13-05-42" src="https://github.com/user-attachments/assets/a1cca970-c1b0-403d-bd63-6a0b9db14607">

**今天，空洞调查协会正式向外界公布：包括HAND对空六课课长星见雅在内的数位六课成员，将受邀参与协会最新型VR系统的战斗测试。协会发言人称，虚狩级别的顶尖数据，将成为系统进一步进化的宝贵力量。
本次测试活动将于光映广场的HIA俱乐部举行，活动日期也将在近日公布。
（※图片为数日前六课抓获恶性空洞犯罪团伙千面』后准备乘车离开时的照片。当时，闻讯而来的粉丝与记者几乎挤满了街道。)**

**Comment by junstar20123** [comment]
支持

**Comment by ZionHuang28** [comment]
怎么已经有人睡在广场排队了

**Comment by chenyunji** [comment]
好！

**Comment by qawdertgy** [comment]
╰（‵□′）╯

**Comment by fzdqz** [comment]
在现场，快被不知道哪来的人挤死了

**Comment by Stormlocken** [comment]
我是雅课长的🐶

**Comment by aomengxin** [comment]
智齿

**Comment by Doubudou45** [comment]
还好我有传送点(´∀｀)

**Comment by swyh0648** [comment]
《千面》被逮捕了，难道《法厄同》也

**Reply by Jaques81** [reply]
千面好像是不知道因为整了什么大活，才被抓的吧？法厄同也没干过什么坏事，对空部多少睁一只眼闭一只眼吧。毕竟把神网搞掉对他们自己也没好处。

**Comment by Asuka-2001** [comment]
支持
